helm/alfresco-content-services/values.yaml

```diff
@@ -115,6 +138,9 @@ alfresco-repository:
     # -- Specify a storageClass for dynamic provisioning
     accessModes:
       - ReadWriteMany
+  environment:
+    JAVA_OPTS: >-
+      -Dmail.password=$MAIL_PASSWORD
```

Comment:
not sure I like it, because if the user need to provide their own JAVA_OPTS needs to keep the default ones

if we need it, can we keep it where it was? (or somewhere else but not in the values like if is worth changing it from the user)